### PR TITLE
Release version 1.2.15

### PR DESCRIPTION
**Pull request by hakontro**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,9 @@
+# 1.2.15
+
+## Fixes
+
+* Fixes a bug where not all datapoints would be read if a timeseries had infrequent datapoints.
```
Comment (wjoel) [inline CHANGELOG.md]: "data points", "time series" is how we write it.

**Comment by Tapped** [comment]
Should we also just fix the issue where we are not setting the limit for Raw? It will improve perf a lot I assume.

**Comment by hakontro** [comment]
> Should we also just fix the issue where we are not setting the limit for Raw? It will improve perf a lot I assume.

Feel free to sneak it in here if it's a quick fix, but I'd like to get this released today.

**Comment by Tapped** [comment]
Should be a quick fix, low effort and very high value

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,9 @@
+# 1.2.15
+
+## Fixes
+
+* Fixes a bug where not all data points would be read if a time series had infrequent data points.
```
Comment (wjoel) [inline CHANGELOG.md]: Maybe we should be more specific about how infrequent they had to be for this bug to occur?

**Comment by hakontro** [comment]
Note: I'm disabling one of the time series tests, since an old time series has been stuck in time series limbo. Will re-enable tomorrow to see if the issue resolves by itself.

**Pull request merged**